### PR TITLE
Create stay-in-touch.css, stay-in-touch.html

### PR DESCRIPTION
**Pull request by duylong13**

### CODE REVIEW

File: Practice2/src/partials/stay-in-touch-section.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<section class="page-stay-in-touch">
+  <h5 class="stay-in-touch-heading">Stay In Touch</h5>
+  <div class="stay-in-touch-container-common">
+    <picture class="stay-in-touch-img">
+      <source media="(min-width: 1440px)" srcset="./assets/images/iphone.png">
+      <img src="./assets/images/iphone.png" alt="iphone" class="iphone-img">
+    </picture>
+    <div class="stay-in-touch-container-primary">
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/partials/stay-in-touch-section.html]: This is a form. We should implement form: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form

File: Practice2/src/styles/components/stay-in-touch.css [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+  .btn-submit {
+    margin: 19px auto;
+  }
+
+  .search-common {
+    max-width: 114px;
+  }
+
+  ::placeholder {
```
Comment (vuongnguyen-agilityio) [inline Practice2/src/styles/components/stay-in-touch.css]: Should add placeholder for specific item. Should not set for all item like that.

**Review by vuongnguyen-agilityio — CHANGES_REQUESTED** [review]

File: Practice2/src/styles/components/stay-in-touch.css (hunk 2 above)
Comment (duylong13) [reply Practice2/src/styles/components/stay-in-touch.css]: Fixed

File: Practice2/src/partials/stay-in-touch-section.html (hunk 1 above)
Comment (duylong13) [reply Practice2/src/partials/stay-in-touch-section.html]: Fixed

**Pull request merged**